src/style.css

```diff
@@ -1,3 +1,88 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+}
+
+@media screen and (max-width: 1200px) {
+  .header {
+    width: 1200px;
+  }
+}
+
+@media screen and (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+
+.img__logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+  margin-left: 50px;
+}
+
+nav {
+  display: inline-block;
+  margin-right: 50px;
+}
+
+.nav__link {
+  font-size: 12px;
+  font-style: normal;
+  line-height: 60px;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin-right: 20px;
+  position: relative;
+  padding: 22px 0 23px;
+}
+
+.nav__link::after {
+  opacity: 0;
+  content: "";
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  position: absolute;
+  bottom: -1px;
+  left: 0;
+  border-radius: 8px;
+}
+
+.nav__link:last-child {
+  margin: 0;
+}
+
+.nav__link:link {
+  color: #000;
+}
+
+.nav__link:visited {
+  color: #000;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
```suggestion
  color: #00acdc;
```
make variables for colors which uses more than once

Reply:
<img width="617" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/f067b28f-70a7-4ea6-8a46-087c2c4fd512">
still not fixed